### PR TITLE
resolves P3 evidence

### LINKED ISSUE #1031: p3 supporting evidence page
https://www.figma.com/file/gzgsQIGQbO1wXZQzEHXhoe/Sprint-3?node-id=567%3A409

image

### PR DESCRIPTION
**Pull request by kevinyang06**
added supporting evidence page

> Please include a summary of the change. List any dependencies that are required for this change (for example, any changes to `package.json`).

Fixes #1031

> Please delete options that are not relevant.

- [x] New feature

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made any needed changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

# Required followup work
configure continue and go back button

### CODE REVIEW

File: f2/src/EvidencePage.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                                    <Trans id="evidencePage.detail3" />
+                                </Li>
+                            </Ul>
+                        </Box>
+
+                        <EvidenceInfoForm
+                            onSubmit={data => {
+                                dispatch({
+                                    type: 'saveFormData',
+                                    data: { scammerDetails: data },
```
Comment (sastels) [inline f2/src/EvidencePage.js]: We should put the data in `evidence`, ie `data: { evidence: data },`

File: f2/src/forms/EvidenceInfoForm.js [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+                                                <Trans id="evidencePage.removeFileButton" />
+                                            </Button>
+                                        </Stack>
+                                    </InfoCard>
+                                </React.Fragment>
+                            ))}
+                        </Container>
+
+                        <NextAndCancelButtons>
+                            <Trans id="evidencePage.nextButton">Next: Impact of scam</Trans>
```
Comment (sastels) [inline f2/src/forms/EvidenceInfoForm.js]: You should just use `<Trans id="evidencePage.nextButton" />` on this line.

File: f2/src/locales/en.json [hunk 3]
```diff
@@ -165,5 +164,16 @@
...
   "locationinfoPage.nextPage": "Next, you will be asked for your contact details.",
+  "evidencePage.title": "Supporting evidence",
+  "evidencePage.intro": "It’s okay if you didn’t keep any documentation of the event. You can skip to the next section.",
+  "evidencePage.details": "Think about including things such as:",
+  "evidencePage.detail1": "Screenshots of webpages, conversations, or email threads.",
+  "evidencePage.detail2": "Images of receipts, transations, or packing slips.",
+  "evidencePage.detail3": "Recording of the demands made by the suspect.",
+  "evidencePage.fileDescription": "Describe what this file shows",
+  "evidencePage.addFileButtom": "Add file",
+  "evidencePage.removeFileButton": "Remove file",
```
Comment (sastels) [inline f2/src/locales/en.json]: This should be "Delete file"

File: f2/src/forms/EvidenceInfoForm.js [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+                                                onClick={() => removeFile(index)}
+                                            >
+                                                <Trans id="evidencePage.removeFileButton" />
+                                            </Button>
+                                        </Stack>
+                                    </InfoCard>
+                                </React.Fragment>
+                            ))}
+                        </Container>
+
```
Comment (sastels) [inline f2/src/forms/EvidenceInfoForm.js]: You should add a `<P>` block with the "Next, you will be asked about your location." here.

File: f2/src/EvidencePage.js (hunk 1 above)
Comment (kevinyang06) [reply f2/src/EvidencePage.js]: fixed

File: f2/src/forms/EvidenceInfoForm.js (hunk 2 above)
Comment (kevinyang06) [reply f2/src/forms/EvidenceInfoForm.js]: fixed

File: f2/src/locales/en.json (hunk 3 above)
Comment (kevinyang06) [reply f2/src/locales/en.json]: fixed

File: f2/src/forms/EvidenceInfoForm.js (hunk 4 above)
Comment (kevinyang06) [reply f2/src/forms/EvidenceInfoForm.js]: fixed

**Comment by sastels** [comment]
Awesome! Now just some new merge conflicts to resolve and then 🚀

**Pull request merged**